### PR TITLE
added search command for quotes

### PR DESCRIPTION
**Pull request by mikytron123**

### CODE REVIEW

File: NadekoBot.Core/Modules/Utility/QuoteCommands.cs [hunk 1]
```diff
@@ -111,6 +111,8 @@ public async Task QuoteShow(int id)
                 await ShowQuoteData(quote);
             }
 
+
+
```
Comment (Gibstick) [inline NadekoBot.Core/Modules/Utility/QuoteCommands.cs]: remove this spurious diff

File: NadekoBot.Core/Services/Database/Repositories/Impl/QuoteRepository.cs [hunk 2]
```diff
@@ -48,6 +48,20 @@ public async Task<Quote> SearchQuoteKeywordTextAsync(ulong guildId, string keywo
                 .FirstOrDefault();
         }
 
+        public IEnumerable<Quote> SearchQuoteKeywordKeyTextAsync(ulong guildId, string keyword)
+        {
+            var q = _set.AsQueryable()
+                .Where(x => x.GuildId == guildId
+                            //&& q.Keyword == keyword
```
Comment (Gibstick) [inline NadekoBot.Core/Services/Database/Repositories/Impl/QuoteRepository.cs]: remove commented code

File: NadekoBot.Core/Services/Database/Repositories/Impl/QuoteRepository.cs [hunk 3]
```diff
@@ -48,6 +48,20 @@ public async Task<Quote> SearchQuoteKeywordTextAsync(ulong guildId, string keywo
...
+                            && EF.Functions.Like(x.Keyword.ToUpper(), $"%{keyword.ToUpper()}%")
+                            // && q.Text.Contains(text, StringComparison.OrdinalIgnoreCase)
```
Comment (Gibstick) [inline NadekoBot.Core/Services/Database/Repositories/Impl/QuoteRepository.cs]: same

File: src/NadekoBot/data/strings/commands/commands.en-US.yml [hunk 4]
```diff
@@ -2040,4 +2040,8 @@ economy:
 purgeuser:
   desc: "Purge user from the database completely. This includes currency, xp, clubs that user owns, waifu info"
   args:
-    - "@Oblivion"
\ No newline at end of file
+    - "@Oblivion"
+quotesearchkey:
+  desc: "Searches for quote using keyword 15 Per page."
+  args:
+    - "test"
```
Comment (Gibstick) [inline src/NadekoBot/data/strings/commands/commands.en-US.yml]: Can you give a better name to this arg, like "keyword" or "query"

**Review by Gibstick — COMMENTED** [review]
Just some superficial fixes

File: NadekoBot.Core/Modules/Utility/QuoteCommands.cs (hunk 1 above)
Comment (mikytron123) [reply NadekoBot.Core/Modules/Utility/QuoteCommands.cs]: fixed

File: NadekoBot.Core/Services/Database/Repositories/Impl/QuoteRepository.cs (hunk 2 above)
Comment (mikytron123) [reply NadekoBot.Core/Services/Database/Repositories/Impl/QuoteRepository.cs]: fixed

File: NadekoBot.Core/Services/Database/Repositories/Impl/QuoteRepository.cs (hunk 3 above)
Comment (mikytron123) [reply NadekoBot.Core/Services/Database/Repositories/Impl/QuoteRepository.cs]: fixed

File: src/NadekoBot/data/strings/commands/commands.en-US.yml (hunk 4 above)
Comment (mikytron123) [reply src/NadekoBot/data/strings/commands/commands.en-US.yml]: changed to keyword

**Pull request merged**